pkg/testing/integration/program.go

```diff
@@ -1748,11 +1758,7 @@ func (pt *ProgramTester) preparePythonProject(projinfo *engine.Projinfo) error {
 		}
 
 		if err := pt.runVirtualEnvCommand("virtualenv-pip-install",
-			[]string{"pip", "install", "-r", "requirements.txt"}, cwd); err != nil {
-			return err
-		}
-	} else {
-		if err = pt.preparePythonProjectWithPipenv(cwd); err != nil {
+			[]string{"python", "-m", "pip", "install", "-r", "requirements.txt"}, cwd); err != nil {
```

Comment:
I changed this here to match what we do inside the Python SDK, running it as `python -m pip install` rather than `pip install`.

https://github.com/pulumi/pulumi/blob/066507cd009ae188db7a2930ef35c1f8b02fade6/sdk/python/python.go#L231